mysql-test/suite/versioning/t/foreign.test

```diff
@@ -266,4 +266,58 @@ delete from a;
 
 drop table b, a;
 
+--echo ###############################################
+--echo # CASCADE UPDATE foreign not system versioned #
+--echo ###############################################
+create or replace table author (
+	id smallint unsigned not null auto_increment,
+	name varchar(100) not null,
+	age int unsigned not null,
+	primary key (id, age)
+) engine = innodb;
+
+--replace_result $sys_datatype_expl SYS_DATATYPE
+eval create or replace table book (
+  id mediumint unsigned not null auto_increment primary key,
+  title varchar(200) not null,
+  author_id smallint unsigned not null,
+  author_age int unsigned not null,
+  sys_start $sys_datatype_expl as row start invisible,
+  sys_end $sys_datatype_expl as row end invisible,
+  period for system_time(sys_start, sys_end),
+  constraint `fk_book_author`
+    foreign key (author_id, author_age) references author (id, age)
+    on delete cascade
+    on update cascade
+) engine = innodb with system versioning;
```

Comment:
Minimize test case. Are all these fields needed? I would use same table names `parent`, `child` like in previous cases.

Reply:
Fixed.